cryptocurrency-subdomain/src/main/kotlin/com/ajax/cryptocurrency/application/ports/repository/CryptocurrencyRepository.kt

```diff
@@ -0,0 +1,20 @@
+package com.ajax.cryptocurrency.application.ports.repository
+
+import com.ajax.cryptocurrency.domain.CryptocurrencyDomain
+import org.springframework.data.domain.Pageable
+import org.springframework.stereotype.Repository
+import reactor.core.publisher.Flux
+import reactor.core.publisher.Mono
+
+@Repository
+interface CryptocurrencyRepository {
```

Comment:
let's call it CryptocurrencyRepositoryOutPort to highlight that it is output port from application to infrastructure

Reply:
done